db/document.go

```diff
@@ -836,7 +844,7 @@ func (doc *Document) IsChannelRemoval(revID string) (bodyBytes []byte, history R
 	}
 	history = encodeRevisions(revHistory)
 
-	return bodyBytes, history, channels, true, isDelete, nil
+	return bodyBytes, history, activeChannels, true, isDelete, nil
```

Comment:
Is this changing the function's API. It looks like it used to return removed channels only here.

Reply:
This was intentional, but the commit message and the PR description don't provide any detail about why this particular change was made.  Code comments would probably also be helpful.

Reply:
👍 In that case, the variable names in `revCacheLoaderForDocument` also need changing 😄

Reply:
Variable names in revCacheLoaderForDocument has been refactored. Also added function level comments.